regression/scripts/AVL/avl_files/results_case_001_01.txt

```diff
@@ -1,53 +0,0 @@
- ---------------------------------------------------------------
```

Comment:
Did you mean to delete all of these results?

Reply:
Yes I did. They have been updated